classes/class-nets-easy-confirmation.php

```diff
@@ -107,6 +107,9 @@ public function confirm_order() {
 	public function maybe_confirm_customer_redirected_from_payment_page_order() {
 
 		$payment_id = filter_input( INPUT_GET, 'paymentId', FILTER_SANITIZE_FULL_SPECIAL_CHARS );
+		if ( empty( $payment_id ) ) {
+			$payment_id = WC()->session->get( 'dibs_payment_id' );
+		}
```

Comment:
Reading the comment for this method it says this:

> 	 The main reason for this is when a purchase is done on mobile phone with a non default browser.
	 Bank ID/Vipps/Swish might then redirect the customer back to the stores checkout page,
	 but in the default browser. **In this scenario it doesn't exist a cart session in WooCommerce.**

Since a cart session does not exist that also means that the a WooCommerce session does not exist, making this fallback unnecessary. It could also in a worst case scenario cause us to get the wrong payment id if the customer for any reason had a live session on the store in two separate browsers. My suggestion would be to just return this to how it was before, and remove the extra step to get the payment id from the session.